doc/README/Data/Tree/AVL.agda

```diff
@@ -28,6 +28,21 @@ open import Relation.Binary.PropositionalEquality
 open Data.Tree.AVL <-strictTotalOrder renaming (Tree to Tree′)
 Tree = Tree′ (MkValue (Vec String) (subst (Vec String)))
 
+-- The first argument to MkValue should be a function from a key
+-- (a ℕ in this case) to a value type (a String vector type).
+-- [Since (Vec String) is missing Vec's second parameter, a ℕ,
+-- it's the required kind of function.]
```

Comment:
I would say, "it's equivalent to the function `\lambda n -> Vec String n`" (with appropriate unicode) rather than "it's the required kind of function". I'd also take the square brackets away entirely.